### DISCUSSION TITLE
Running in GitHub Actions

### DISCUSSION THREAD

**Discussion by SSP6904** [benjamincburns/websockproxy#22] [Q&A]
Hi!

I'm trying to do this in my v86 repo, along with the Docker file in the root.
But it takes a while or if that's just me..

Is there a version that can deploy on GitHub actions?

**Comment by benjamincburns** [comment]
I'm not sure what you mean by "deploy" in this context. That said, I think you'd maybe benefit from using [a third party docker-compose action](https://github.com/marketplace/actions/docker-compose-action).

Unfortunately I expect that if you were to try this it would fail, as currently the websockproxy container needs to run in privileged mode so that it can allocate TUN/TAP resources. I don't think it'd be terribly difficult to modify it to avoid the TUN/TAP device stuff and have it only switch packets between connected clients, however. I'll create a new feature issue to capture this.

**Reply by benjamincburns** [reply]
I created #23 to account for this. @SSP6904 - please have a look through what I've proposed there and let me know whether that will work for your use case by replying to that issue.

**Reply by SSP6904** [reply]
I'll be honest with you, I totally forgot about this issue, and I haven't been using this for a while now. I'm all good with it for now.
